_docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/topic/field--node--field-data-tags--topic.html.twig

```diff
@@ -0,0 +1,56 @@
+{#
+/**
+ * @file
+ * Default theme implementation for a field.
+ *
+ * To override output, copy the "field.html.twig" from the templates directory
+ * to your theme's directory and customize it, just like customizing other
+ * Drupal templates such as page.html.twig or node.html.twig.
+ *
+ * Instead of overriding the theming for all fields, you can also just override
+ * theming for a subset of fields using
+ * @link themeable Theme hook suggestions. @endlink For example,
+ * here are some theme hook suggestions that can be used for a field_foo field
+ * on an article node type:
+ * - field--node--field-foo--article.html.twig
+ * - field--node--field-foo.html.twig
+ * - field--node--article.html.twig
+ * - field--field-foo.html.twig
+ * - field--text-with-summary.html.twig
+ * - field.html.twig
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the containing element.
+ * - label_hidden: Whether to show the field label or not.
+ * - title_attributes: HTML attributes for the title.
+ * - label: The label for the field.
+ * - multiple: TRUE if a field can contain multiple items.
+ * - items: List of all the field items. Each item contains:
+ *   - attributes: List of HTML attributes for each item.
+ *   - content: The field item's content.
+ * - entity_type: The entity type to which the field belongs.
+ * - field_name: The name of the field.
+ * - field_type: The type of the field.
+ * - label_display: The display settings for the label.
+ *
+ * @see template_preprocess_field()
+ *
+ * @ingroup themeable
+ */
+#}
+
+{% set values = items[0]["content"]["#context"]["value"]|split(',') %}
+{% set data = "[" %}
+
+{% for item in values %}
+    {% if not loop.last %}
+        {% set data = data ~"'"~ item|trim ~"'" ~ ',' %}
+    {% else %}
+        {% set data = data ~ "'" ~ item|trim ~ "'" ~ "]" %}
+
+    {% endif %}
+{% endfor %}
+
+<div id="topic-resources" class="row" data-tags={{ data }}>
```

Comment:
It seems like some twig filters would work best here:

http://twig.sensiolabs.org/doc/filters/index.html

I'm thinking `<div id="topic-resources" class="row" data-tags={{content|split(',')|json_encode}}>` would really be all you'd need to do here. I'll try it locally and if it works I'll make the change here.